_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw.cpp

```diff
@@ -182,6 +222,14 @@ mfxStatus SetLowpowerDefault(MfxVideoParam& par)
         return sts;
     }
 #endif // MFX_VERSION >= 1025
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+    if ((par.m_platform == MFX_HW_EHL)
+        && par.mfx.LowPower == MFX_CODINGOPTION_UNKNOWN)
+    {
+        par.mfx.LowPower = MFX_CODINGOPTION_ON;
```

Comment:
Does EHL has only LowPower encoder?

Reply:
This code just change default encoder to be a low-power one. @saosipov could you please comment?

Reply:
In CS we have the following lines instead of added by Oleg:
#if !defined(MFX_CLOSED_PLATFORMS_DISABLE) && (MFX_VERSION >= MFX_VERSION_NEXT)
    if ((par.m_platform == MFX_HW_LKF || par.m_platform == MFX_HW_JSL || par.m_platform == MFX_HW_DG2)
        && par.mfx.LowPower == MFX_CODINGOPTION_UNKNOWN)
    {
        par.mfx.LowPower = MFX_CODINGOPTION_ON;
        return sts;
    }
#endif
#if defined(MFX_ENABLE_HEVCE_SCC)
    if (par.m_platform >= MFX_HW_TGL_LP
        && par.mfx.LowPower == MFX_CODINGOPTION_UNKNOWN
        && par.mfx.CodecProfile == MFX_PROFILE_HEVC_SCC)
    {
        par.mfx.LowPower = MFX_CODINGOPTION_ON;
        return sts;
    }
#endif
As I understand DG2 should not be mentioned in OS only, the rest should be left.
Oleg, could you explain?